Libraries/Text/RCTUITextView.m

```diff
@@ -51,6 +51,25 @@ - (void)dealloc
   [[NSNotificationCenter defaultCenter] removeObserver:self];
 }
 
+- (NSString *)accessibilityLabel
+{
+  NSMutableString *accLabel = [NSMutableString stringWithString:@""];
```

Comment:
Should be `NSMutableString *accessibilityLabel = [NSMutableString new];`.
(Please, use `new` and, please, avoid abbreviation.)